standard/expressions.md

```diff
@@ -3688,6 +3817,55 @@ At run-time, the expression `await t` is evaluated as follows:
 
 An awaiter’s implementation of the interface methods `INotifyCompletion.OnCompleted` and `ICriticalNotifyCompletion.UnsafeOnCompleted` should cause the delegate `r` to be invoked at most once. Otherwise, the behavior of the enclosing async function is undefined.
 
+## §range-operator Range operator
+
+This operator provides a succinct syntax for specifying a (possibly empty) element range suitable for use in denoting a slice of an indexable sequence (§indexable-sequence).
+
+```ANTLR
+range_expression
+    : unary_expression
+    | range_expression? '..' range_expression?
+    ;
+```
+
+For an operation of the form `s .. e`, binary operator overload resolution ([§12.4.5](expressions.md#1245-binary-operator-overload-resolution)) is applied to select a specific operator implementation. The operands are converted to the parameter types of the selected operator, and the type of the result is the return type of the operator. *range_expression* shall have type `System.Index` or a type that can be converted implicitly to that type. Only one predefined range operator exists:
```

Comment:
```suggestion
For an operation of the form `s .. e`, binary operator overload resolution ([§12.4.5](expressions.md#1245-binary-operator-overload-resolution)) is applied to select a specific operator implementation. The operands are converted to the parameter types of the selected operator, and the type of the result is the return type of the operator. Each of `s` and `e` shall have type `System.Index` or a type that can be converted implicitly to that type. Only one predefined range operator exists:
```

(Or let's clarify in a different way. The fact that `range_expression` uses `range_expression` make this more confusing.)